### PR TITLE
Add review checklist, contributing file, update readme

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Add review checklist, contributing file, update readme

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review
cc @cognifloyd could you please take a look as a native speaker?

File: README.md [hunk 1]
```diff
@@ -120,6 +131,10 @@ ansible-test integration --docker default -v --color --python 3.6
 ansible-test units --docker default -v --color --python 3.6
 ```
 
+### Collection maintenance
+
+Maintainers of this collection follow the [mainteiner guidelines](https://github.com/ansible/community-docs/blob/main/maintaining.rst).
```
Comment (odyssey4me) [inline README.md:136]: mainteiner <-- spelling error

Comment (Andersson007) [reply README.md]: @odyssey4me thanks! Fixed

**Review by odyssey4me — APPROVED** [review]

**Comment by Andersson007** [comment]
@odyssey4me thanks for reviewing!

**Pull request merged**